content/en/docs/tutorials/security/apparmor.md

```diff
@@ -119,6 +119,10 @@ The output should look something like this:
 cri-containerd.apparmor.d (enforce)
 ```
 
+## Ephemeral containers
+
+ The `container.apparmor.security.beta.kubernetes.io/*` annotations don't apply to ephemeral containers. When an ephemeral container runs in [privileged mode](/docs/concepts/workloads/pods/#privileged-mode-for-containers), the kubelet does not apply any AppArmor restrictions to the ephemeral container. Otherwise, the kubelet will attempt to run the ephemeral container with the default runtime profile.
```

Comment:
Please wrap the lines
```suggestion
 The `container.apparmor.security.beta.kubernetes.io/*` annotations don't apply to ephemeral containers.
 When an ephemeral container runs in [privileged mode](/docs/concepts/workloads/pods/#privileged-mode-for-containers), the kubelet does not apply AppArmor restrictions to the ephemeral container.
Otherwise, the kubelet will attempt to run the ephemeral container with the default runtime profile.
```